Greenheart/swipeforfuture.com
# fix(deps): Upgrade to React 17, CRA 4 and TS 4

One breaking change with `tsconfig.json`, but other than that our project upgraded really smoothly. Looking forward to test the new CRA dev tools more closely!

The fact that three major upgrades could be this simple really makes it clear how far these technologies have come in terms of stability compared to a few years back. Good times.

To test:
1. `rm -rf node_modules`
2. `npm install`
3. `npm start`

Then everything should work as expected.
